packages/components/src/progress-bar/README.md

```diff
@@ -1,13 +1,42 @@
 # ProgressBar
 
-<div class="callout callout-alert">
-This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
-</div>
-
 A simple horizontal progress bar component.
 
 Supports two modes: determinate and indeterminate. A progress bar is determinate when a specific progress value has been specified (from 0 to 100), and indeterminate when a value hasn't been specified.
 
+## Usage
+
+Basic usage:
+
+```jsx
+import { ProgressBar } from '@wordpress/components';
+
+const MyLoadingComponent = () => {
+  return <ProgressBar />
+}
```

Comment:
When declaring a constant, you also need a semicolon at the end:

```suggestion
};
```

Won't be necessary if we use the hoisted function syntax.

Reply:
Yeah, my bad; I don't have syntax high-checking for jsx code blocks, so this slipped through the cracks. Thanks for spotting those.